src/routes.rs

```diff
@@ -560,14 +562,8 @@ pub async fn get_collections_children(
     let content_type = get_content_type_from_headers(req.headers_mut());
 
     // We dont listen to pagination. We have a hard max of 250 per collection
-    let mut limit: i32 = 250;
-    let mut offset: i32 = 0;
-
-    // in we dont remove watched then we dont need to limit
-    if config.include_watched {
```

Comment:
this can be adapted and kept. As it enables pagination (not supported when using excluded watched)